homeassistant/components/lovelace/dashboard.py

```diff
@@ -251,10 +252,14 @@ async def _async_load_data(self) -> collection.SerializedStorageCollection | Non
 
     async def _process_create_data(self, data: dict) -> dict:
         """Validate the config is valid."""
-        if "-" not in data[CONF_URL_PATH]:
+        url_path = data[CONF_URL_PATH]
+
+        allow_single_word = data.pop(CONF_ALLOW_SINGLE_WORD, False)
```

Comment:
If we pop it out, it means we don't save it. This means the user cannot rename this panel without updating the URL?

Reply:
And also it's not possible to update the url 😅

Reply:
URL can not be changed according to the frontend and the update schema : 
```py
DASHBOARD_BASE_UPDATE_FIELDS = {
    vol.Optional(CONF_REQUIRE_ADMIN): cv.boolean,
    vol.Optional(CONF_ICON): vol.Any(cv.icon, None),
    vol.Optional(CONF_TITLE): cv.string,
    vol.Optional(CONF_SHOW_IN_SIDEBAR): cv.boolean,
}
```